src/utils/validation/registration/registrationValidation.ts

```diff
@@ -87,14 +86,6 @@ export const registrationValidationSchema = Yup.object<YupShape<Registration>>({
       }
     }),
   }),
-  associatedArtifacts: Yup.array()
```

Comment:
Nå mister vi validering på lisens og embargo-dato også da 🤔 (legg til fil, og gå til annet panel uten å legge til lisens skal trigge feilmelding for lisens)
Skulle kanskje vært noe sånt: `associatedArtifacts: Yup.array().of(associatedFileValidationSchema)`?